app/api/video_stream.py

```diff
@@ -125,6 +125,29 @@ def create_bbb_meeting(channel, data):
     data['extra'] = res.data
 
 
+@streams_routes.route(
+    '/<int:stream_id>/getRecordings',
```

Comment:
```suggestion
    '/<int:stream_id>/recordings',
```

And only organizer should be able to access it